### PR TITLE
Tweak home UI

### PR DESCRIPTION
**Pull request by lovincyrus**
- move difficulty to the top
- tweak tags

image

### CODE REVIEW

**Review by nassdonald — APPROVED** [review]

**Review by okwme — APPROVED** [review]
i like it!

**Comment by okwme** [comment]
i've seen these bulds failing
do we need new setting in our CI?

File: .vuepress/components/Home.vue [hunk 1]
```diff
@@ -169,6 +170,26 @@ a
...
+        font-size: 0.875rem;
+        line-height: 1.25rem;
+        letter-spacing: 0.03em;
+        text-transform: capitalize;
+        background: var(--tag-background-color);
+        border-radius: 0.25rem;
+        padding: 0.25rem 0.5rem 0.125rem;
+        width: fit-content;
+      }
+
```
Comment (okwme) [inline .vuepress/components/Home.vue:190]:
```suggestion
```

Comment (okwme) [reply .vuepress/components/Home.vue]: (just to trigger a rebuild to see if marko's changes to CI fix this or caused this)

**Comment by okwme** [comment]
looks like it was the broken before and now it's fixed

**Pull request merged**